opcua/common/manage_nodes.py

```diff
@@ -85,7 +85,13 @@ def create_variable(parent, *args):
     """
```

Comment:
I think we should move that code to me method create _ variable _ type

On Wed, Jul 6, 2016, 07:59 Denis Štogl <notifications@github.com> wrote:

> In opcua/common/manage_nodes.py
> <https://github.com/FreeOpcUa/python-opcua/pull/235#discussion_r69678825>:
>
> > @@ -85,7 +85,13 @@ def create_variable(parent, *args):
> >      """
>
> Add to comment: "[data type]"
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FreeOpcUa/python-opcua/pull/235/files/20a394e891394054faff3f2ed1b3d7548f59a13c#r69678825>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACcfzqmlsBIijGdUuitn4dWDYZfSosocks5qS0Q1gaJpZM4JFcMH>
> .
>